Cabal/src/Distribution/Simple/InstallDirs.hs

```diff
@@ -506,6 +512,12 @@ instance Read PathTemplate where
     , (template, "") <- reads path
     ]
 
+instance Parsec PathTemplate where
+  parsec = parsecPathTemplate
+
+parsecPathTemplate :: CabalParsing m => m PathTemplate
+parsecPathTemplate = parsecFilePath >>= return . toPathTemplate
```

Comment:
```suggestion
parsecPathTemplate = parsecFilePath <&> toPathTemplate
```

```
$ hlint Cabal/src/Distribution/Simple/InstallDirs.hs
Cabal/src/Distribution/Simple/InstallDirs.hs:519:22-63: Suggestion: Use <&>
Found:
  parsecFilePath >>= return . toPathTemplate
Perhaps:
  parsecFilePath Data.Functor.<&> toPathTemplate
```

This will also need the import from `Data.Functor` of `(<&>)`.